_config.yml

```diff
@@ -49,7 +49,6 @@ frogs:         "[Ryukyufrogs](https://www.ryukyu-frogs.com)"
 
 highlighter: rouge
 markdown: kramdown
```

Comment:
Same here, already the default, not needed.

Reply:
Didn't know it! Thanks for your comments! 😻✨

Reply:
See https://github.com/jekyll/jekyll/blob/master/lib/jekyll/configuration.rb#L7-L78 for the whole default config options.